# Remove C++ code from a C header; was a workaround for older Intel compilers.
# It was for a macro that simply suppresses unused variable warnings
--- geometry-0.3.5/geometry.orig/src/qhull_a.h	2014-11-07 21:37:50.000000000 +0100
+++ geometry-0.3.5/geometry/src/qhull_a.h	2015-01-08 13:02:17.000000000 +0100
@@ -103,9 +103,7 @@
 #   define QHULL_OS_WIN
 #endif
 #if defined(__INTEL_COMPILER) && !defined(QHULL_OS_WIN)
-template <typename T>
-inline void qhullUnused(T &x) { (void)x; }
-#  define QHULL_UNUSED(x) qhullUnused(x);
+#  define QHULL_UNUSED(x) (x);
 #else
 #  define QHULL_UNUSED(x) (void)x;
 #endif
